runtime/calamari/src/xcm_config.rs

```diff
@@ -129,7 +128,7 @@ parameter_types! {
     /// Used in native traders
     /// This might be able to skipped.
     /// We have to use `here()` because of reanchoring logic
-    pub ParaTokenPerSecond: (xcm::v2::AssetId, u128) = (Concrete(MultiLocation::here()), 1_000_000_000);
+    pub ParaTokenPerSecond: (cumulus_primitives_core::AssetId, u128, u128) = (Concrete(MultiLocation::here()), 1_000_000_000, 0);
```

Comment:
Maybe add a todo or task to implement this properly at some point. the mb/sec parameter.
It will have to be changed in the FirstAsseetTrader impl as well.

Reply:
#1274